config/HELICSConfig.cmake.in

```diff
@@ -64,7 +64,9 @@ if (HELICS_HAVE_ZEROMQ)
 	if (TARGET libzmq)
 	else()
 		set(ZeroMQ_LIBRARY_ONLY ON)
-		set(ZeroMQ_INSTALL_PATH ${PACKAGE_PREFIX_DIR})
+		if(NOT ZeroMQ_INSTALL_PATH)
```

Comment:
We might want to think about handling environment variables here as well since we are now doing that in some of the other things dealing with ZMQ location.  Same thing with BOOST

Reply:
I think Boost might be okay since it is using the addBoost.cmake module which uses the environment variable as a possible search location.